tests/Acceptance/TokenEndpointTest.php

```diff
@@ -154,7 +158,8 @@ public function testSuccessfulAuthorizationCodeRequestWithPublicClient(): void
         $jsonResponse = json_decode($response->getContent(), true);
 
         $this->assertSame('Bearer', $jsonResponse['token_type']);
-        $this->assertSame(3600, $jsonResponse['expires_in']);
+        $this->assertLessThanOrEqual(3600, $jsonResponse['expires_in']);
+        $this->assertGreaterThan(0, $jsonResponse['expires_in']);
```

Comment:
why no assertSameWithDelta?

Reply:
It's because I didn't want `3601` to be valid